drivers/i2c/Kconfig

```diff
@@ -93,41 +93,41 @@ endif # I2C_RTIO
 
 # Include these first so that any properties (e.g. defaults) below can be
 # overridden (by defining symbols in multiple locations)
+source "drivers/i2c/Kconfig.ambiq"
```

Comment:
Might as well add `zephyr-keep-sorted-start` and `zephyr-keep-sorted-stop` to keep them sorted

Reply:
Will do for this file (if PR is acceptable) after a rebase, the CMakeLists.txt is a bit tricky as the Kconfig (`CONFIG_..` precedes filenames), I think sorting based on the file names looks cleaner.